blueocean-dashboard/src/main/js/AllPipelines.jsx

```diff
@@ -0,0 +1,14 @@
+import React, { Component } from 'react';
+import OrganisationPipelines from './OrganisationPipelines';
+
+class AllPipelines extends Component {
+
+    render() {
+        return (
+            <OrganisationPipelines { ... this.props } />
+        );
+    }
+
+}
+
+export default AllPipelines;
```

Comment:
Yeah, originally I was going to use these higher level components to set the `fetchPipelines` and `fetchPipelinesIfNeeded` properties of `OrganisationPipelines`, with different action funcs that would invoke a different REST URL. It requires quite a bit of plumbing, and I realized it made sense just to update the existing action functions with a `organizationName` property that would cause one URL or the other to be called. It's clear now the higher-level components are doing so little that I can just drive the logic back into `OrganisationPipelines` because it's so simple.